Hymns/Display/DisplayHymnBottomBar.swift

```diff
@@ -30,16 +31,24 @@ struct DisplayHymnBottomBar: View {
         HStack(spacing: 0) {
             Group {
                 Spacer()
-                Button(action: {}, label: {
-                    BottomBarLabel(imageName: "square.and.arrow.up")
-                })
+                    Button(action: {
+                        self.showShareSheet = true
+                    }, label: {
+                        BottomBarLabel(imageName: "square.and.arrow.up")
+                    })
+                .sheet(isPresented: $showShareSheet) {
+                    ShareSheet(activityItems: [self.viewModel.sharablelLyrics])
+                }
```

Comment:
I would put this sheet on the HStack, so all that logic is in the same place and better organized.

Also, please use `.sheet(item: Binding<Identifiable?>, content: (Identifiable) -> View)` instead of `.sheet(isPresented: Binding<Bool>, content: () -> View)`

and create a 
```
enum DisplayHymnSheet: String {
    case share = "Share Lyrics"
}
```
and switch on that, kind of like what I did with `DisplayHymnActionSheet`.

The rationale here is that, there could potentially be 7 (or more) different bottom sheets. But there are only 3 **types** of bottom sheets. This reduces the number of @State variables we'll have to manage, by only managing one state variable per **type** of bottom sheet, instead of one per action.

Reply:
Nice, makes sense. resolved in most recent commit